ios/Sources/AutoLayoutView.swift

```diff
@@ -46,7 +51,15 @@ import UIKit
     /// Tracks where first pixel is drawn in the visible window
     private var lastMinBound: CGFloat = 0
 
-    override func layoutSubviews() {
+    private func getSubviews() -> [UIView] {
```

Comment:
nit (if this can be overridden – or we can name this property differently):

```suggestion
    private override var subviews: [UIView] {
```

Reply:
I tried to override it but it's being used internally by UIKit and was causing exceptions. I renamed it to `viewsToLayout`, is that an ok name?